app/client/pages/user/home.js

```diff
@@ -0,0 +1,114 @@
+import { Field, Form, Formik } from "formik";
```

Comment:
homepage should be at `/` not at `/user/home`. Right now, `/` sends people to registration, but change that to open this page.
Other note add we need to add icon and app name and other stuff as nav bar on top. I am not going to block the PR since this is out of scope. I just want to call that out here.